src/harness/sas_interface.py

```diff
@@ -200,9 +200,21 @@ def BlacklistByFccIdAndSerialNumber(self, request):
     pass
 
   @abc.abstractmethod
+  def InjectEscZone(self, request):
+    """SAS admin interface to inject EscZone information into SAS under test.
+     
+    Args:
+      request: key-value pair where the key is
+        "zone" and the value is  he information of the zone of type ZoneData as specified in specified in SAS-SAS TS:
```

Comment:
Can we use "zoneData" to be consistent with other methods?

Let's try:
... "zoneData" and the value is ZoneData structure as defined in the SAS-SAS TS (WINNF-16-S-0096).